### ISSUE TITLE
Transfer Apple Live Photos to Android with Localsend

### ISSUE THREAD

**Issue by shuke12306** [localsend/localsend#1580]
I transferred photos taken on my iPhone to Pixel via Localsend and backed them up to Google Photos. I've been using it like this for six months, and suddenly one day the live photos transferred to the Pixel won't show up live in Google Photos anymore, and they've turned into regular static photos. So I would like to ask if this problem is caused by Localsend? Can the devs fix this issue? Thanks.

**Comment by bboyler** [comment]
Yes, localsend removes live photo, very frustrating when I found out

**Comment by NightMean** [comment]
There is already a feature request to add this from 2023 but there is no progress on it

See https://github.com/localsend/localsend/issues/1052

**Comment by shuke12306** [comment]
I have been transferring livephoto from my iPhone to my Android device via localsend for the last year, but suddenly one day it failed. I feel very puzzled.
> There is already a feature request to add this from 2023 but there is no progress on it
> 
> 
> 
> See https://github.com/localsend/localsend/issues/1052
